app/models/user.rb

```diff
@@ -21,6 +21,12 @@ def weekly_habits
       .order('habit_entries.created_at DESC')
   end
 
+  def monthly_habits(month)
+    habit_entries
+      .where('extract(month from habit_entries.created_at) = ? AND habit_entries.status = 1', month)
+      .order('habit_entries.created_at DESC')
+  end
```

Comment:
This query does not have the group clause that the other habit entry one does, in fact the other one doesn't need it either (at least all tests passed without it).